src/platform/imx8/platform.c

```diff
@@ -145,6 +145,8 @@ int platform_init(struct sof *sof)
 	clock_init();
 	scheduler_init_edf(sof);
 
+	platform_interrupt_init();
```

Comment:
What do you think about making it the first call in the function? It would be good precaution in case `clock_init `or `scheduler_init_edf` will use your interrupts in the future.

Reply:
@paulstelian97 @tlauda  makes sense to me.